code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -823,7 +823,7 @@
 		M.KnockDown(1)
 
 /datum/reagent/xeno_neurotoxin/on_overdose(mob/living/M)
-		M.adjustToxLoss(volume * 0.2 * REM) //Overdose starts applying more toxin damage
+		M.adjustOxyLoss(volume * 0.2 * REM) //Overdose starts applying more oxy damage
```

Comment:
Tying it to `volume` is a bit too punishing. Consider this happens on overdose, so `volume` will be really high here.